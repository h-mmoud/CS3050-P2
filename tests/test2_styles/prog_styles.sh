#!/bin/bash
sh ../run.sh ../examples/styles.fp