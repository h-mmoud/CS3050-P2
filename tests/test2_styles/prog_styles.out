hastings
lawrence

X = lawrence, Y = cynthia